src/main/drivers/accgyro/accgyro_spi_lsm6dsv16x.c

```diff
@@ -0,0 +1,267 @@
+/*
+ * This file is part of Cleanflight and Betaflight.
```

Comment:
New files should use new license header

 https://raw.githubusercontent.com/betaflight/betaflight/master/DEFAULT_LICENCE.md

Reply:
Fixed.